packages/agents/cartographer/poller/db/migrations/20220824094332_metrics_transfer_add_fee_revenue.sql

```diff
@@ -0,0 +1,130 @@
+-- migrate:up
+
+-- 
+-- Schema change
+-- 
+ALTER TABLE public.transfers
+ADD COLUMN IF NOT EXISTS transfer_status_update_by_agent numeric;
+
+ALTER TABLE public.transfers
+ADD COLUMN IF NOT EXISTS transfer_status_message_by_agent numeric;
+
+-- 
+-- New Hourly Metrics
+-- 
+-- 1. Transfer metrics
+CREATE  OR REPLACE VIEW public.hourly_transfer_metrics AS (
```

Comment:
if possible, Can you add the API links in the doc and create a new example file for hourly metrics commands?

Reply:
added.